### PR TITLE
[stable/rabbitmq]: adds podLabels to values.yaml

### PR DESCRIPTION
**Pull request by logicbomb421**
Signed-off-by: Michael Hill <mhill@betterview.net>

#### What this PR does / why we need it:
This PR adds the ability to add custom pod labels to the pods created in this chart. This can be useful when defining custom network policies that need to be applied to consumed charts.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 3.5.5
```
Comment (desaintmartin) [inline stable/rabbitmq/Chart.yaml:2]: Bumb a minor version since this is a new feature.

**Review by juan131 — COMMENTED** [review]
Thanks for the PR! As @desaintmartin mentioned, please bump the minor version!

**Comment by logicbomb421** [comment]
@juan131 @desaintmartin minor version bumped!

**Pull request merged**